pootle/checks.py

```diff
@@ -45,6 +46,27 @@ def _version_to_string(version, significance=None):
 
 
 @checks.register()
+def check_duplicate_emails(app_configs=None, **kwargs):
+    from accounts.utils import get_duplicate_emails
+    errors = []    
+    try:
+        if len(get_duplicate_emails()):
+            errors.append(checks.Warning(_("There are user accounts with "
+                                           "duplicate emails. This will not "
+                                           "be allowed in Pootle 2.8."),
+                                         hint=_("Try using 'pootle "
+                                                "find_duplicate_emails', and "
+                                                "then update user emails with "
+                                                "'pootle update_user_email "
+                                                "username email'"),
+                                         id="pootle.W017"))
+    except (OperationalError, ProgrammingError):
```

Comment:
im not sure about this - but i concluded that this error should only ever happen in tests when the checks are run - so i figured that was the most appropriate place for it.